cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractAccountApiTest.java

```diff
@@ -151,8 +151,10 @@ public void testDeleteSubAccount() throws Exception {
     @Test
     public void testGetUser() throws Exception {
         ApiResponse user = createUser();
-        ApiResponse result = account.user(user.get("id").toString(), null);
+        String id = user.get("id").toString();
+        ApiResponse result = account.user(id, null);
         assertNotNull(result);
+        deleteUser(id);
```

Comment:
Please move all `deleteUser` calls to `tearDown`, if assertion fails, this line is never called.